### ISSUE TITLE
Disable keybind feature

### ISSUE THREAD

**Issue by u0398** [Auroali/armour-bundles#1]
It would be nice to be able to disable the keybind/switching feature. That way the bundles could be used just as armor storage.

**Comment by Auroali** [comment]
that wouldn’t be too difficult to implement, would it be better to have it be a gamerule or config option?

**Comment by u0398** [comment]
I don't think it would effect server fairness, and is more of a player preference so a configuration may be better.
